contrib/win32/win32compat/win32_pty.c

```diff
@@ -131,7 +131,7 @@ int exec_command_with_pty(int * pid, char* cmd, int in, int out, int err, unsign
 	}
 
 	debug3("pty commandline: %ls", pty_cmdline);
-	if (CreateProcessW(NULL, pty_cmdline, NULL, NULL, TRUE, 0, NULL, NULL, &si, &pi)) {
+	if (CreateProcessW(NULL, pty_cmdline, NULL, NULL, TRUE, CREATE_NO_WINDOW, NULL, NULL, &si, &pi)) {
```

Comment:
this change should not be in this pull request.
this change is not complete: it is not the pty host that needs `CREATE_NO_WINDOW`, it is the subprocess sshd.exe spawned to handle the connection.